### PR TITLE
ENH log all file removals

### PR DESCRIPTION
**Pull request by sidneymau**

### CODE REVIEW

**Comment by beckermr** [comment]
Is there a reason you want error here as opposed to warning or info? There are a lot of files so I could see this as debug too.

**Comment by sidneymau** [comment]
No reason, just made it consistent with the other logging messages for deleting the files.

File: eastlake/steps/delete_images.py [hunk 1]
```diff
@@ -65,13 +66,16 @@ def execute(self, stash, new_params=None):
                         if (seg_file is not None):
                             if os.path.isfile(seg_file):
                                 self.logger.error("removing file %s" % seg_file)
+                                os.remove(seg_file)
```
Comment (sidneymau) [inline eastlake/steps/delete_images.py:69]: This change is also more of a BUG than an ENH—there was a logging statement about removing a file but no call to actually remove that file.

Comment (beckermr) [reply eastlake/steps/delete_images.py:69]: Arg woops.

**Review by beckermr — APPROVED** [review]
Lgtm!

**Comment by sidneymau** [comment]
Before merging, do we want to make all of the log calls for individual files as something other than `error`? We can leave the messages about deleting a whole category as is

**Comment by beckermr** [comment]
Yeah those should be debug.

**Pull request merged**